onnxruntime/core/providers/cuda/cuda_call.cc

```diff
@@ -82,14 +90,14 @@ bool CudaCall(ERRTYPE retCode, const char* exprString, const char* libName, ERRT
 #endif
       int currentCudaDevice;
       cudaGetDevice(&currentCudaDevice);
+      cudaGetLastError();  // clear last CUDA error
```

Comment:
Scott McKay
If we're clearing the error regardless of whether we throw on not, line 101 isn't required anymore.

Sherlock Huang
good catch